lib/elixir/src/elixir_compiler.erl

```diff
@@ -85,15 +85,15 @@ code_mod(Fun, Expr, Line, File, Module, Vars) when is_binary(File), is_integer(L
   Tuple = {tuple, Line, [{var, Line, Var} || {_, Var} <- Vars]},
```

Comment:
I should probably change this line as well. I'm wondering if there are other spots that are not getting caught by dialyzer like this one.